Add README

narcissusTheFlower opened the pull request:
Trying out new README

narcissusTheFlower committed:
Create README.md

narcissusTheFlower committed:
Update README.md

narcissusTheFlower committed:
Update README.md

narcissusTheFlower committed:
Update README.md

narcissusTheFlower committed:
Update README.md

narcissusTheFlower committed:
Update README.md

narcissusTheFlower committed:
Update README.md

narcissusTheFlower committed:
Update README.md

narcissusTheFlower committed:
Update README.md

maximilionus commented on README.md:
```diff
@@ -0,0 +1,59 @@
+2) Download the [latest archive](https://github.com/maximilionus/byedpictl/archive/refs/heads/master.zip) and unpack it
+3) In the unpacked directory run `sudo ./install.sh`
+
+<hr>
```
We don't need the line break here, it's already handled by renderer on proper headers.

maximilionus commented on README.md:
```diff
@@ -0,0 +1,59 @@
+
+<hr>
+  
+<h3>Usage</h3>
```
Wrong header level, it should be `<h2>` in our case. This will also fix the styling, as the second level headers also have the underline, which makes them easier for visual navigation.

To make it even better, replace it with Markdown syntax:

```
## Usage
```

maximilionus commented on README.md:
```diff
@@ -0,0 +1,59 @@
+
+See the list of possible commands with `./bydpictl help`
+  
+<hr>
```
The same as above, wrong level header below and no need for line break here

maximilionus commented on README.md:
````diff
@@ -0,0 +1,59 @@
+--auto=torst --timeout=3"
+```
+
+<hr>
````
Same as above

maximilionus commented on README.md:
```diff
@@ -0,0 +1,59 @@
+<h1>Running byedpi and socks5 proxy together</h1>
```
That header is way tooooooooooooooooooooooooooo long. Would be much better just to replace it with just project name, the description below explains the details.

maximilionus reviewed (commented):
Yep, this one looks much better, thank you.

The only thing left is a few syntax inconsistencies in file. You use the Markdown, but apply the html tags in it. It's not restricted, but very non-preferred: the only real need to use them is something like header-less tables or hidden details, something outside of what basic Markdown can handle. You can follow [this guide](https://www.markdownguide.org/basic-syntax/) to get all the info about converting the html tags to Markdown.

Let me know if you are willing to apply the suggested improvements. I can take it from here if you are short on time.

narcissusTheFlower commented:
> Let me know if you are willing to apply the suggested improvements. I can take it from here if you are short on time.

Yeah, I can apply the changes

narcissusTheFlower committed:
Applying markdown tags

I tried to remove undesired linebreaks, tho, I still left some for the sake of plain text readability

maximilionus commented:
Already looks much better after d4191a4bcec69556739442890f8b25fb5bc5b53a. Mark the PR as "Ready for review" when ready, so I can merge it.

maximilionus commented:
Thank you :heart:
